RecoEgamma/EgammaTools/plugins/EGRegressionModifierV3.cc

```diff
@@ -231,8 +236,10 @@ std::array<float,32> EGRegressionModifierV3::getRegData(const reco::GsfElectron&
     data[26] = iEta;
     data[27] = iPhi;
     data[28] = (iEta-signIEta)%5;
-    data[29] = (iPhi-1)%2;
-    data[30] = (std::abs(iEta)<=25)*((iEta-signIEta)) + (std::abs(iEta)>25)*((iEta-26*signIEta)%20);  
+    data[29] = (iPhi-1)%2; 
+    const int iEtaCorr = iEta - (iEta > 0 ? +1 : -1);
+    const int iEtaCorr26 = iEta - (iEta > 0 ? +26 : -26);
```

Comment:
just to refresh my memory: is it true that iEta can not be equal to 0?

Reply:
yes, it can not be true for a valid EB DetId.  It goes directly from -1 to 1